include/boost/histogram/detail/span.hpp

```diff
@@ -8,7 +8,8 @@
 #define BOOST_HISTOGRAM_DETAIL_SPAN_HPP
 
```

Comment:
```suggestion
#if __has_include(<version>)
#include <version>
#endif
```

Something like this, perhaps?

Reply:
Yes